app/controllers/smart_proxies_controller.rb

```diff
@@ -24,7 +27,6 @@ def edit
   end
 
   def ping
-    @proxy = @smart_proxy
```

Comment:
Looks like no view uses that in Foreman, but are you sure there is no plugin using that member variable? Had to have a reason.